src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -61,37 +64,48 @@ class FastSync(
   }
 
   def startWithState(syncState: SyncState): Unit = {
-    if (syncState.updatingTargetBlock) {
-      log.info(s"FastSync interrupted during targetBlock update, choosing new target block")
+    if (syncState.updatingPivotBlock) {
+      log.info(s"FastSync interrupted during pivot block update, choosing new pivot block")
       val syncingHandler = new SyncingHandler(syncState)
-      val targetBlockSelector = context.actorOf(FastSyncTargetBlockSelector.props(etcPeerManager, peerEventBus, syncConfig, scheduler), "target-block-selector")
-      targetBlockSelector ! FastSyncTargetBlockSelector.ChooseTargetBlock
-      context become syncingHandler.waitingForTargetBlockUpdate(ImportedLastBlock)
+      val pivotBlockSelector = context.actorOf(
```

Comment:
Does it make sense maybe to create `PivotBlockSelector` actor once and ask him for pivot block whenever there's a need for that?

Reply:
I am not sure. I don't want to change that in this PR. Will see after fastsync refactoring what solution will be the best